### ISSUE TITLE
error: Cannot read property 'https://localhost/mgmt/tm/sys/ready/0' of undefined

### ISSUE THREAD

**Issue by ghalevy** [F5Networks/f5-bigip-runtime-init#16]
When deploying a larger (20+) number of concurrent instances (terraform) I'll usually get one or two nodes fail runtime-init with the following (or similar) errors.  Manually triggering the binary works fine:

```
[admin@localhost:NO LICENSE:Standalone] ~ # cat /var/log/cloud/bigIpRuntimeInit.log 
2021-04-09T04:13:12.582Z [27324]: info: Configuration file: /config/cloud/runtime-init-conf.yaml
2021-04-09T04:13:12.589Z [27324]: info: Validating provided declaration
2021-04-09T04:13:12.653Z [27324]: info: Successfully validated declaration
2021-04-09T04:13:12.653Z [27324]: info: Executing custom pre_onboard_enabled commands
2021-04-09T04:13:14.108Z [27324]: info: Shell command: /usr/bin/setdb provision.extramb 2048 execution completed; response: 
2021-04-09T04:13:14.264Z [27324]: info: Shell command: /usr/bin/setdb restjavad.useextramb true execution completed; response: 
2021-04-09T04:29:55.241Z [27324]: error: Cannot read property 'https://localhost/mgmt/tm/sys/ready/0' of undefined
2021-04-09T04:29:55.242Z [27324]: info: Sending F5 Teem report for failure case.
```

runtime-init code:

```
---
runtime_parameters: []
pre_onboard_enabled:
  - name: provision_rest
    type: inline
    commands:
      - /usr/bin/setdb provision.extramb 2048
      - /usr/bin/setdb restjavad.useextramb true
extension_packages:
  install_operations:
  %{~ for extension_name, schema_version in SCHEMA_VERSIONS ~}
    - extensionType: ${extension_name}
      extensionVersion: ${schema_version}
  %{~ endfor ~}
```

_NOTE: runtime-init is executed by Azure Custom-Script provider._

Cloud: Azure
TMOS: 15.1.2
f5-bigip-runtime-init pkg: f5-bigip-runtime-init-all-1.2.0-1-signed.noarch.rpm

**Comment by shyawnkarim** [comment]
Thanks for reporting this. We are now tracking this issue internally with ID ESECLDTPLT-2567.

**Comment by shyawnkarim** [comment]
Closing.

This bug fix was included with [Release 1.2.1](https://github.com/F5Networks/f5-bigip-runtime-init/releases/tag/1.2.1).
